bigbluebutton-html5/imports/ui/components/presentation/styles.js

```diff
@@ -131,8 +131,6 @@ const PresentationContainer = styled.div`
   left: 0;
   right: 0;
   bottom: 0;
-  ${({ isVideoFocus, isRTL }) => isVideoFocus && isRTL && `padding: 0px 0px 0px ${contentSidebarMarginToMedia}`};
```

Comment:
These two were introduced in https://github.com/bigbluebutton/bigbluebutton/pull/22204
Could you please confirm we're not regressing in RTL support?

Reply:
Yes, I tested and it was good also in rtl, seems it was needed before but now it was misplacing it.